### DISCUSSION TITLE
Libre 2 without MiaoMiao

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#2104] [General]
Can someone point me to the best instructions for pairing a Libre 2 sensor to xDrip ?   I started mine with a Libre reader, but can't get xDrip to read the transmitted data. 

Thanks

**Comment by Navid200** [comment]
See if you can find what you are looking for here:
https://xdrip.readthedocs.io/en/latest/install/prerequisites/

**Reply by GLoren1** [reply]
Thanks.  Among that documentation , I read "Only Libre 2 Europe sensor model is supported" from a 4-28-2022 update.   Do you know whether that is accurate?

**Reply by Navid200** [reply]
@psonnera

**Reply by psonnera** [reply]
Yes. Only Libre 2 European version is supported. Libre 2 US, Canada and Australia use a different protocol.

**Comment by GLoren1** [comment]
Thanks for the info.  Are there any plans to add support for the the US/Canada version protocol?

**Reply by psonnera** [reply]
We don't have an easy solution. This is the only workaround now: https://xdrip.readthedocs.io/en/latest/install/nightscoutfollower/#diabox
